Source/WTF/wtf/PlatformEnableCocoa.h

```diff
@@ -139,7 +139,7 @@
 
 #endif // !PLATFORM(MAC) && !PLATFORM(MACCATALYST) && !PLATFORM(WATCHOS) && !PLATFORM(APPLETV)
 
-#if !defined(ENABLE_APPLE_PAY_SESSION_V11) && ((PLATFORM(MAC) && __MAC_OS_X_VERSION_MIN_REQUIRED >= 110300) || (PLATFORM(IOS) && __IPHONE_OS_VERSION_MIN_REQUIRED >= 140500) || PLATFORM(VISION))
```

Comment:
Yep. The plan of attack is to do different platforms in different patches.

Reply:
(The new plan of attack is to do this year-by-year instead of year-by-year and platform-by-platform. Will start that after landing this current batch of patches.)